src/moaicore/MOAIFreeTypeFont.cpp

```diff
@@ -343,10 +343,10 @@ int MOAIFreeTypeFont::_setDefaultSize(lua_State *L){
 FT_Face MOAIFreeTypeFont::AffirmFreeTypeFace () {
 	
 	if (!this->mFreeTypeFace) {
-		FT_Library library;
-		FT_Error error = FT_Init_FreeType( &library );
+		this->mLibrary = new FT_Library();
+		FT_Error error = FT_Init_FreeType( this->mLibrary );
 		CHECK_ERROR(error);
-		return this->LoadFreeTypeFace( &library );
+		return this->LoadFreeTypeFace( this->mLibrary );
```

Comment:
Wow, I just started digging down method calls and almost lose sanity.
I mean, passing around pointers to protected fields so that other methods store stuff in them? C is crazy.

Anyhoo, based on "There is exactly one library object instance per MOAIFreeTypeFont instance." my concern is gone.